postcss.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+
+module.exports = (ctx) => {
+    return {
+        plugins: [
+            require('postcss-fonticons')({
+                iconPath: path.resolve(__dirname, 'assets/icons/'),
+            }),
+            require('postcss-preset-env')({
+                browsers: ['last 2 versions', '> 5%'],
```

Comment:
This is duplicated from the `package.json`:

https://github.com/datenanfragen/website/blob/9195f52c9366844e6fe8c08fe88952776eb29c05/package.json#L114-L117

Since this is a JS file, can't we just import that here? Or, if the p.json entry isn't needed anymore, we should remove that.

But this is not a blocking problem, I'll create an issue.

Reply:
@baltpeter, should we consider removing it?